### PR TITLE
Updated introduction email

### PR DESCRIPTION
**Pull request by philon-**

### CODE REVIEW

**Review by emanuelen5 — APPROVED** [review]
Grymt!

File: api/src/templates/new_member.body.html [hunk 1]
```diff
@@ -52,38 +52,37 @@ <h3>Vad är mitt medlemsnummer, är min tagg aktiv och hur lång tid till är ta
...
+<p>Yes! Köp din egen förvaringsbox med måtten 50x39x26 cm (32L). Märk den med en etikett från <a href="https://wiki.makerspace.se/Memberbooth">Memberbooth</a> och ställ den på en en fri plats på hyllplanen i korridoren. Lådan finns ex vis på <a href="https://www.clasohlson.com/se/F%C3%B6rvaringsl%C3%A5da-SmartStore-Home/34-1552-31">Clas Ohlson</a>. </p>
 
 <h3>Jag bygger något större än labblådan, var ska jag förvara det?</h3>
-<p>I mån av plats så kan du lägga ditt projekt i hall-hyllorna i max 3 månader. Ditt projekt ska vara märkt med datum, medlemsnummer och namn, annars försvinner det. Vi har plats för köpta skivor av plywood, akryl och liknande mittemot laserskäraren i workshop-rummet.</p>
+<p>I mån av plats så kan du lägga ditt projekt i hall-hyllorna i max 3 månader. Ditt projekt ska vara märkt med en etikett från <a href="https://wiki.makerspace.se/Memberbooth">Memberbooth</a>, annars försvinner det. Vi har plats för köpta skivor av plywood, akryl och liknande brevid laserskäraren i stora rummet. Alla regler kring förvaring på Stockholm Makerspace går att hitta på <a href="https://wiki.makerspace.se/Medlemsf%C3%B6rvaring">wikin.</a></p>
```
Comment (HalfVoxel) [inline api/src/templates/new_member.body.html:64]:
```suggestion
<p>I mån av plats så kan du lägga ditt projekt i hall-hyllorna i max 3 månader. Ditt projekt ska vara märkt med en etikett från <a href="https://wiki.makerspace.se/Memberbooth">Memberbooth</a>, annars försvinner det. Vi har plats för köpta skivor av plywood, akryl och liknande bredvid laserskäraren i stora rummet. Alla regler kring förvaring på Stockholm Makerspace går att hitta på <a href="https://wiki.makerspace.se/Medlemsf%C3%B6rvaring">wikin.</a></p>
```

**Review by HalfVoxel — CHANGES_REQUESTED** [review]
Nice! Just a tiiny typo.

**Review by HalfVoxel — APPROVED** [review]
Took the liberty of fixing the typo.

**Pull request merged**